azure-pipelines.yml

```diff
@@ -192,11 +192,18 @@ jobs:
       compatibility/update-versions.sh
       git add compatibility/versions.bzl compatibility/maven_install.json
 
-      BRANCH=update-compat-versions-for-$(release_tag)-$(Build.BuildId)
-      TITLE="update compat versions for $(release_tag)"
-      open_pr "$BRANCH" "$TITLE"
 
-      setvar "branch" "$BRANCH"
+      if ! git diff --cached --quiet; then
```

Comment:
not particularly confident in my bash in Yaml skills so please review carefully

Reply:
What about pulling it out in its own script, testing it and calling the script from here?

Reply:
it is somewhat intertwined with the conditions on the Azure config which is the part I’m really worried about. Pulling it out seems to make those links harder to see and I still can’t test the part that I’m worried about.